commit 0482504290d9d2f065dd8cb0d0c697c788724db3
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Sep 21 10:29:38 2009 +0300

    Remove XA "support" from the backend
    - it's removed from BDB 4.8.x and this has hardly ever done anything anyway
    (cherry picked from commit 510c6763ef745b308abd1cc6cec2a62119e87c9c)

diff --git a/lib/backend/dbconfig.c b/lib/backend/dbconfig.c
index e668036..2b16241 100644
--- a/lib/backend/dbconfig.c
+++ b/lib/backend/dbconfig.c
@@ -40,9 +40,6 @@ struct poptOption rdbOptions[] = {
 	NULL, NULL },
 #endif
 
- { "xa_create",	0,POPT_BIT_SET,	&db3dbi.dbi_cflags, DB_XA_CREATE,
-	NULL, NULL },
-
  { "create",	0,POPT_BIT_SET,	&db3dbi.dbi_oeflags, DB_CREATE,
 	NULL, NULL },
  { "thread",	0,POPT_BIT_SET,	&db3dbi.dbi_oeflags, DB_THREAD,
